docs/api/en/objects/SkinnedMesh.html

```diff
@@ -153,6 +153,11 @@ <h3>[method:null updateMatrixWorld]( [param:Boolean force] )</h3>
 		Updates the [page:Matrix4 MatrixWorld].
 		</p>
 
+		<h3>[method:Vector3 boneTransform]( [index:Integer], [target:Vector3] )</h3>
+		<p>
+		Calculates the position of the vertex at the given index, based on the 1-4 bones that may influence that vertex.
```

Comment:
I think I would not mentioned the number of possible bones. That is technical detail which should not mentioned at that place.